package.json

```diff
@@ -2,9 +2,9 @@
   "name": "locales-check",
```

Comment:
[sug] we can use a scoped name like ```@mok-labs/locales-check``` to reduce the chance of name conflicts

Reply:
The name locales-check wasn't used in npm, and I understand that thery are unique, so if it is not used we shouldn't have name conflicts

Reply:
Scoped names could be useful to [prevent some attacks](https://github.blog/2021-02-12-avoiding-npm-substitution-attacks/) and signal ownership

_Each npm user/organization has their own scope, and only you can add packages in your scope. This means you don't have to worry about someone taking your package name ahead of you. Thus it is also a good way to signal official packages for organizations._

[scoped packages from npmjs.com](https://docs.npmjs.com/cli/v10/using-npm/scope)

Reply:
Interesting, thanks! I'll create an organization then and change the name

Reply:
done! ✅

Reply:
wait, I forgot updating the readme 😬

Reply:
now its updated ✅